v2/test/azuresql_test.go

```diff
@@ -0,0 +1,373 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package test
+
+import (
+	"testing"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql" //sql drive link
```

Comment:
it imports for side-effects, see https://stackoverflow.com/questions/21220077/what-does-an-underscore-in-front-of-an-import-statement-mean

It's a common thing w/ sql helpers.

With that said, we don't actually need this one here, because this is AzureSQL not MySQL. Swapped this to the correct import (It's not technically needed because it's imported in helpers we call but putting it here too I think helps clarify/convey we're using the SQL driver here.

Doing this actually made me realize theres now a MSFT fork of the mssql driver, so updated to use that too. Great catch.